test/device/test_accelerometer.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+from app.device.accelerometer import Accelerometer
+
+
+def test_not_moving():

```

Comment:
lets also add some tests for potential error cases and make sure we fail gracefully in weird cases
- all 0s in the register
- all 1s in the register